src/components/Table.md

```diff
@@ -671,11 +671,7 @@ const Example = () => {
 
   return (
     <Card width="100%">
-      <Table
-        items={items.slice(startIndex, endIndex + 1)}
-        getItemKey={item => item.id}
-        columns={columns}
-      />
+      <Table items={[]} getItemKey={item => item.id} columns={columns} />
```

Comment:
Why are we not passing in the `items` to the Table component here?

Reply:
Because it's a markdown example where the user simply adds items instead of selecting from existing ones